src/Controller/SubmitController.php

```diff
@@ -25,11 +25,12 @@ public function redirectAction(Request $request) : Response
             ];
             $subRequest = $request->duplicate(null, null, $path);
             $subRequest->headers->set('Referer', $request->getUri());
+            $subRequest->getSession()->set('journal.submit', true);
```

Comment:
I guess there is no way to pass this as a parameter given the redirects flow involved

Reply:
Could read better as `'journal.new_session', true`?

Reply:
Used `submit` to refer to it being triggered by the `SubmitController`. `new_session` would always be true!